core/lib/multivm/src/interface/mod.rs

```diff
@@ -0,0 +1,12 @@
+pub mod types;
+
+pub use types::{
```

Comment:
Nit: IMHO, it's better to expose `errors`/`inputs`/`outputs` rather than all the types in one pack.

Reply:
Agree, it was done like this before, don't want to make a lot of changes